js/stellarnav.js

```diff
@@ -224,6 +198,11 @@
 			// expands the dropdown menu on each click of nav-text
 			nav.find('li .sub-list').on('click', navbarExpand);
 
+			nav.find('li.nav-item').not('.has-sub').on('click', (e) => {
+				toggleMenu(e);
+				window.location.href = e.target.href;
```

Comment:
I'm not certain why `e.preventDefault()` was originally added, so to avoid causing any breaking changes, I introduced `window.location.href = e.target.href` to ensure navigation to the intended URL works